core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/RequestManager.java

```diff
@@ -604,4 +495,28 @@
   public void addPoolWatcher(Pool pool, User watcher) throws IOException;
 
   public void removePoolWatcher(Pool pool, User watcher) throws IOException;
+
+  @Deprecated
+  long saveLibrary(Library library) throws IOException;
+
+  @Deprecated
+  long saveLibraryQC(LibraryQC libraryQc) throws IOException;
+
+  @Deprecated
+  public LibraryType getLibraryTypeByDescriptionAndPlatform(String description, PlatformType platformType) throws IOException;
+
+  @Deprecated
+  public LibrarySelectionType getLibrarySelectionTypeByName(String name) throws IOException;
+
+  @Deprecated
+  public LibraryStrategyType getLibraryStrategyTypeByName(String name) throws IOException;
+
+  @Deprecated
+  public long saveLibraryDilution(LibraryDilution libraryDilution) throws IOException;
+
+  @Deprecated
+  public Collection<Library> listAllLibrariesBySampleId(long sampleId) throws IOException;
+
+  @Deprecated
+  public Collection<LibraryDilution> listAllLibraryDilutions() throws IOException;
```

Comment:
The FormUtils methods that use RequestManager could go into a different, autowirable class which has the appropriate Services wired into it. It'd be fair to call that out of scope of this ticket though